### ISSUE TITLE
Created a Cassandra Flavour - enjoy

### ISSUE THREAD

**Issue by BlueHotDog** [hiddentao/squel#265]
Hey all,
I've created a cassandra CQL flavour for Squel, let me know what you think..

Also, maybe it's a good idea to extract all of the flavours to separate repos?

https://github.com/getjaco/squel_cassandra

**Comment by hiddentao** [comment]
Awesome work! 

I like the idea of extracting the flavours into separate repos. I think squel need a bit of re-architecture to provide a better way of registering and loading in flavours.

**Comment by BlueHotDog** [comment]
Yeah, i also thought about doing some work on making the flavours easier to use, mainly the fact that it changes the global squel object

What are your thoughts regarding this?

**Comment by hiddentao** [comment]
Actually in the current version using a flavour builds a new squel object, so the default object remains unaffected. Thus you can use multiple flavours in the same codebase.

**Comment by BlueHotDog** [comment]
ah, awesome! than i'm pretty happy with it as it's :)

BTW, found another small issue with the way you handle valueHandlers, see the change here:
https://github.com/getjaco/squel_cassandra/blob/master/src/blocks/where_block.js#L35

Basically, some value handlers might want to do the identity function..

**Comment by hiddentao** [comment]
Yeah, that was a recent change I did due to inconsistency in how custom formatted values were being handled later on in the process. I've simply pushed responsibility upto whoever implements the custom value format handler. If you want to do the identity function then it's upto you to take into account how that value may be rendered in output.
